docs/file_types/parquet_files.md

```diff
@@ -0,0 +1,47 @@
+# Parquet files you might encounter throughout PIXL
+
+## OMOP-ES files
+
+From
+[OMOP-ES](https://github.com/UCLH-Foundry/the-rolling-skeleton/blob/main/docs/design/100-day-design.md#data-flow-through-components)
+we receive parquet files defining the data we need to export. These input files appear as 2 groups:
+
+1. **Public** parquet files: have had identifiers removed and replaced with a sequential ID for the
+   export
+2. **Private** parquet files: map sequential identifiers to patient identifiers (e.g. MRNs,
+   Accession numbers, NHS numbers)
+
+## Radiology reports
+
+In addition to the OMOP-ES files, the PIXL pipeline generates **Radiology** parquet files, which
+contain the radiology reports for the given extract. These are generated by collecting all
+_de-identified_ radiology reports for the given extract from the CogStack API.
```

Comment:
```suggestion
The PIXL pipeline generates **Radiology** parquet files, which
contain the radiology reports for the given extract. These are generated by calling the CogStack API, which returns a de-identified radiology report given a full radiology report.
```

Clarify that the radiology reports go *through* Cogstack (ie not the new way where they'll be sitting waiting for us). Also it sounded a bit like PIXL generates OMOP ES files.